submissions/medinatapanpa/html/dufuna.html

```diff
@@ -2,23 +2,23 @@
 <html lang="en">
 <head>
         <title>Registration Page</title>
+        <link rel="stylesheet" href="dufuna.css">
 </head>
 <body>
- <img src="pistachios.jpg" alt="medicine" width="60" height="40">
- <article width="40" >
+
+ <img src="pistachios.jpg" alt="herbs" class= "pics">
+ <article class="article" >
   Herbals is a Nigerian based herbal online store founded in 1984. Our herbal house sells various kinds of 
   herbal spieces and medicine. Herbals offers a diverse option of herbs, from Nigerian, Mediterrenian and American 
  to our customers. Customers experience an open option to diverse blends of herbs from all over the world in 
  thier finest flavours.</article>
-<style>
-  form { align-self: auto;
-    }
-</style>
- <P>
+
+ <P>  
+   <div class="bigbox">
     <form style="align-content: space-between;">
```

Comment:
Please move inline style definition to your stylesheet for easy maintainability.

Reply:
I have done that. thank you